language/vm/src/constant.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
I am not clear this should be in vm but it's fine for now.
Any thoughts on this?

Reply:
I'm fine having this in vm right now. The reason is that `Constant` is a file format related concept, thus whenever an upstream is trying to make sense of the constants in the serialized modules it should import the `Constant` in `file_format`.